librz/asm/arch/hexagon/hexagon_disas.c

```diff
@@ -8,7 +8,13 @@
 
 extern ut32 constant_extender;
 
-int hexagon_disasm_instruction(ut32 hi_u32, HexInsn *hi, ut32 addr) {
+#if ASAN
```

Comment:
why this?

Reply:
because of (https://github.com/rizinorg/rizin/runs/1608321103#step:12:2082):

![possible_OOM](https://user-images.githubusercontent.com/12002672/103146098-75efca80-477f-11eb-8758-d0aeb76ff99a.PNG)

I'm assuming gcc ran out of memory (based on e.g. https://gcc.gnu.org/bugzilla/show_bug.cgi?id=87316#c3).